Schedule.php

```diff
@@ -42,6 +42,9 @@ public function call($callback, array $parameters = array())
      */
     public function command($command)
     {
+        //get full path of the cliScriptName in linux
+        //ToDo add logic for windows envviroment
+        $this->cliScriptName  =   Yii::getAlias("@app").'/yii';
```

Comment:
1. You should not use `'yii'` here as string constant, developer should be able to specify cli name using `$cliScriptName`

2. `command` method is not the best place for `Yii::getAlias` call, better to move it into `init()`, for example